controllers/getAllTopicsController.js

```diff
@@ -0,0 +1,12 @@
+const getAllTopics = require("../models/getAllTopicsModel");
+
+async function getAllTopicsController(req, res, next) {
+  try {
+    const topics = await getAllTopics();
+    return res.status(200).send(topics.rows);
```

Comment:
The best place to extract the data off the rows key is in the model, as it's data manipulation, which is the model's thing